packages/dds/tree/src/util/index.ts

```diff
@@ -3,8 +3,44 @@
  * Licensed under the MIT License.
  */
 
-export * from "./utils";
-export * from "./typeCheck";
-export * from "./brand";
-export * from "./offsetList";
-export * from "./stackyIterator";
+export {
+    clone,
```

Comment:
Could you sort the exports alphabetically with your tool?  Both the exported members as well as sorting the files themselves.

Reply:
There isn't a way to do it automatically, so I'll have to modify the script. I'll create a separate ticket for it as a future task since the longer this PR takes to merge the more time it takes to constantly rebase it and keep it up to date :)